### PR TITLE
Bumping go build version to 1.15.x

### PR DESCRIPTION
**Pull request by apluchik**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/microsoft/moc
 
-go 1.14
+go 1.15
```
Comment (madhanrm) [inline go.mod:4]: pipeline.yaml changes?

Comment (apluchik) [reply go.mod:4]: The moc/moc-pkg pipelines don't currently have the GoTool task in their build steps. Should I add that as part of this PR?

Comment (madhanrm) [reply go.mod:4]: Yes, lets add it

Comment (apluchik) [reply go.mod:4]: added GoTool task

**Review by madhanrm — APPROVED** [review]

**Pull request merged**